signinapp/model.py

```diff
@@ -240,15 +240,16 @@ def make(
         return user
 
     @staticmethod
-    def make_guardian(name: str, phone_number: str, email: str, contact_order: int = 1):
+    def make_guardian(name: str, phone_number: str, email: str):
         role = Role.from_name("guardian_limited")
+        pn = normalize_phone_number_for_storage(phone_number)
```

Comment:
Is this still necessary?

Reply:
Alas, yes - `get_from` doesn't normalize the phone number, and we use that in `__init__.py`. Plus it's good to be safe.